engine/src/Graphics/Models/ModelManager.cpp

```diff
@@ -0,0 +1,66 @@
+        vertices.push_back({ Vec4f(0.5f, -0.5f,  0.5f, 1.0f),  Vec4f(1.0f, 0.0f, 0.0f, 0.0f),  Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 0.0f) });
+                                 
+        vertices.push_back({ Vec4f(-0.5f,  0.5f, -0.5f, 1.0f), Vec4f(-1.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 1.0f) });
+        vertices.push_back({ Vec4f(-0.5f,  0.5f,  0.5f, 1.0f), Vec4f(-1.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 1.0f) });
+        vertices.push_back({ Vec4f(-0.5f, -0.5f,  0.5f, 1.0f), Vec4f(-1.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 0.0f) });
+        vertices.push_back({ Vec4f(-0.5f, -0.5f, -0.5f, 1.0f), Vec4f(-1.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 0.0f) });
+                                 
+        vertices.push_back({ Vec4f(-0.5f,  0.5f, 0.5f, 1.0f),  Vec4f(0.0f, 0.0f, 1.0f, 0.0f),  Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 1.0f) });
+        vertices.push_back({ Vec4f(0.5f,  0.5f, 0.5f, 1.0f),   Vec4f(0.0f, 0.0f, 1.0f, 0.0f),  Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 1.0f) });
+        vertices.push_back({ Vec4f(0.5f, -0.5f, 0.5f, 1.0f),   Vec4f(0.0f, 0.0f, 1.0f, 0.0f),  Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 0.0f) });
+        vertices.push_back({ Vec4f(-0.5f, -0.5f, 0.5f, 1.0f),  Vec4f(0.0f, 0.0f, 1.0f, 0.0f),  Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 0.0f) });
+                                 
+        vertices.push_back({ Vec4f(0.5f,  0.5f, -0.5f, 1.0f),  Vec4f(0.0f, 0.0f, -1.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 1.0f) });
+        vertices.push_back({ Vec4f(-0.5f,  0.5f, -0.5f, 1.0f), Vec4f(0.0f, 0.0f, -1.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 1.0f) });
+        vertices.push_back({ Vec4f(-0.5f, -0.5f, -0.5f, 1.0f), Vec4f(0.0f, 0.0f, -1.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(0.0f, 0.0f) });
+        vertices.push_back({ Vec4f(0.5f, -0.5f, -0.5f, 1.0f),  Vec4f(0.0f, 0.0f, -1.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(0.0f, 0.0f, 0.0f, 0.0f), Vec4f(1.0f, 1.0f, 1.0f, 1.0f), Vec2f(1.0f, 0.0f) });
```

Comment:
would be nice to add constant variables for tangent, bitangent and color

Reply:
yep!